dev-tools/reconfigurator-cli/tests/input/cmds-expunge-newly-added-external-dns.txt

```diff
@@ -4,9 +4,12 @@ load-example --seed test_expunge_newly_added_external_dns
 
 blueprint-show 3f00b694-1b16-4aaa-8f78-e6b3a527b434
 blueprint-edit 3f00b694-1b16-4aaa-8f78-e6b3a527b434 expunge-zone 9995de32-dd52-4eb1-b0eb-141eb84bc739
+blueprint-diff 3f00b694-1b16-4aaa-8f78-e6b3a527b434 366b0b68-d80e-4bc1-abd3-dc69837847e0
```

Comment:
unfortunately, between the diff size and having conflicting changes on main, i had a hard time keeping the output a more legible "file moved and now has some additional lines". instead, git shows the diff as a fully new file even though it's mostly the prior content.

`blueprint-diff` includes the DNS output though, which is of course what i actually care about here. if this is a bear to review (and i'm pretty empathetic to it being a lot) i'm open to moving the DNS checking over to a new test and leaving this unchanged, or moving the internal DNS testing to live in this test as well.